### ISSUE TITLE
Built-in functions are loaded with both platform and file resource URIs

### ISSUE THREAD

**Issue by kittaakos** [crapo/sadlos2#231]
After some console logging in the `com.ge.research.sadl.jena.JenaBasedSadlModelProcessor.addEquation(Resource, Equation, EObject)` method, one can see that the resource for the built-in functions is loaded once with the file URI and once with the platform URI. That is not the case for tests.

```

resource platform:/resource/AA/C.sadl 0 []
Add equation to resource C.sadl: subtractDates

resource platform:/resource/AA/C.sadl 0 []
Add equation to resource C.sadl: subtractDates
resource file:/Users/akos.kitta/dev/wss/sadl_master/ws/runtime-configuration-assert/AA/ImplicitModel/SadlBuiltinFunctions.sadl 0 []
Add equation to resource SadlBuiltinFunctions.sadl: now

resource file:/Users/akos.kitta/dev/wss/sadl_master/ws/runtime-configuration-assert/AA/ImplicitModel/SadlBuiltinFunctions.sadl 1 [now]
Add equation to resource SadlBuiltinFunctions.sadl: subtractDates

resource platform:/resource/AA/C.sadl 0 []
Add equation to resource C.sadl: subtractDates
resource platform:/resource/AA/ImplicitModel/SadlBuiltinFunctions.sadl 0 []
Add equation to resource SadlBuiltinFunctions.sadl: now

resource platform:/resource/AA/ImplicitModel/SadlBuiltinFunctions.sadl 1 [now]
Add equation to resource SadlBuiltinFunctions.sadl: subtractDates
resource file:/Users/akos.kitta/dev/wss/sadl_master/ws/runtime-configuration-assert/AA/ImplicitModel/SadlBuiltinFunctions.sadl 0 []
Add equation to resource SadlBuiltinFunctions.sadl: now

resource file:/Users/akos.kitta/dev/wss/sadl_master/ws/runtime-configuration-assert/AA/ImplicitModel/SadlBuiltinFunctions.sadl 1 [now]
Add equation to resource SadlBuiltinFunctions.sadl: subtractDates
resource platform:/resource/AA/ImplicitModel/SadlBuiltinFunctions.sadl 0 []
Add equation to resource SadlBuiltinFunctions.sadl: now

resource platform:/resource/AA/ImplicitModel/SadlBuiltinFunctions.sadl 1 [now]
Add equation to resource SadlBuiltinFunctions.sadl: subtractDates

resource platform:/resource/AA/C.sadl 0 []
Add equation to resource C.sadl: subtractDates
resource platform:/resource/AA/ImplicitModel/SadlBuiltinFunctions.sadl 0 []
Add equation to resource SadlBuiltinFunctions.sadl: now

resource platform:/resource/AA/ImplicitModel/SadlBuiltinFunctions.sadl 1 [now]
Add equation to resource SadlBuiltinFunctions.sadl: subtractDates

resource platform:/resource/AA/C.sadl 0 []
Add equation to resource C.sadl: subtractDates

```

**Comment by kittaakos** [comment]
Will be fixed as part of #226.

I will add some logic that uses platform resource URI when the Eclipse platform is running, otherwise, we will fall back to the file-based URIs.

**Comment by kittaakos** [comment]
PR: https://github.com/crapo/sadlos2/pull/234.
